www/js/recommendations.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('emission.main.recommendations',['emission.services', 'emission.plugin.logger',
+                'emission.survey.launch',
+                'ngSanitize', 'ngAnimate',
+                'emission.splash.referral', 'angularLocalStorage',
+                'ng-walkthrough', 'nzTour'])
```

Comment:
nit: you can probably clean up this dependency list since you are not using the functionality in here.
Can defer to cleanup.